packages/cf-adjudicator-contracts/package.json

```diff
@@ -32,7 +32,7 @@
     "solidity"
   ],
   "devDependencies": {
-    "@counterfactual/types": "0.0.40",
+    "@counterfactual/types": "0.0.39",
```

Comment:
Change this to `0.0.41` - if it's `0.0.39` then the next change that bumps this to `0.0.40` will conflict with the current published npm package.